nl_server/search.py

```diff
@@ -89,7 +89,7 @@ def _rank_vars(candidates: EmbeddingsResult,
         sv2sentences[dcid].add(c.sentence)
 
   for sv, score in sorted(sv2score.items(),
-                          key=lambda item: (-item[1], item[0])):
+                          key=lambda item: (-round(item[1], 6), item[0])):
```

Comment:
Maybe have TODO to truncate score using model parameter?

Reply:
Oh also leave a comment why we do it

Reply:
Done